tests/python/frontend/pytorch/test_forward.py

```diff
@@ -379,6 +379,32 @@ def test_forward_maxpool1d():
                                     stride=2).eval(),
                 input_data)
 
+def test_forward_split():
+    torch.set_grad_enabled(False)
+    input_shape = [4, 10]
+
+    class Split1(Module):
+        def forward(self, *args):
+            return torch.split(args[0], 2, 0)
+
+    class Split2(Module):
+        def forward(self, *args):
+            return torch.split(args[0], [2, 3, 5], 1)
+
+    class Split3(Module):
+        def forward(self, *args):
+            return torch.split(args[0], 3, 1)
+
+    class Split4(Module):
+        def forward(self, *args):
+            return torch.split(args[0], 4, 1)
+
```

Comment:
Rather than adding four classes, just add one class with constructor for its two arguments.

Reply:
done.